zephyr/wrapper.c

```diff
@@ -50,18 +50,23 @@ DECLARE_TR_CTX(zephyr_tr, SOF_UUID(zephyr_uuid), LOG_LEVEL_INFO);
 #define HEAP_SYSTEM_SIZE	0
 #endif
 
-/* The Zephyr heap - TODO: split heap */
-#define HEAP_SIZE	(HEAP_SYSTEM_SIZE + HEAP_RUNTIME_SIZE + HEAP_BUFFER_SIZE)
-static uint8_t __aligned(64) heapmem[HEAP_SIZE];
+/* The Zephyr heap */
+#define HEAPMEM_SIZE	(HEAP_SYSTEM_SIZE + HEAP_RUNTIME_SIZE + HEAP_BUFFER_SIZE)
+#define HEAPMEM_SHARED_SIZE	(HEAP_RUNTIME_SHARED_SIZE + HEAP_SYSTEM_SHARED_SIZE)
+
+static uint8_t __aligned(PLATFORM_DCACHE_ALIGN)heapmem[HEAPMEM_SIZE];
+static uint8_t __aligned(PLATFORM_DCACHE_ALIGN)heapmem_shared[HEAPMEM_SHARED_SIZE];
```

Comment:
I think a space after the closing parenthesis would be good :-)

Reply:
@lyakh Actually SOF checkpatch complains about this, so I removed it. This is a type cast here, so the space should not be there (although it looks weird).

Reply:
I don't think it's a type-cast. Checkpatch might've wrongly assumed so. It's an `__attribute__((aligned()))`. I think the "correct" / preferred form is `int x __attribute__((z))` which might also keep checkpatch happy

Reply:
Fixed this now, put the attribute at the end.